include/ppx/xr_component.h

```diff
@@ -168,15 +192,20 @@ class XrComponent
     XrSession  GetSession() const { return mSession; }
     void       SetCurrentViewIndex(uint32_t index) { mCurrentViewIndex = index; }
     uint32_t   GetCurrentViewIndex() const { return mCurrentViewIndex; }
+    XrCamera&  GetCamera();
```

Comment:
const XrCamera& GetCamera() const;?

Reply:
Yes, that's should be in the cleaned up version, this is a draft and I'm still going through the detail of controller input stuff.
`UpdateCamera` probably belong to `BeginFrame` and this function can be constant.